File: packages/sdk-dart/lib/src/tinyhumans_error.dart
class TinyHumansError implements Exception {
  final String message;
  final int status;
  final String body;

  TinyHumansError(this.message, this.status, [this.body = '']);

  @override
  String toString() => 'TinyHumansError($status): $message';
}

File: packages/sdk-csharp/src/TinyHumans.Sdk/TinyHumansError.cs
namespace TinyHumans.Sdk;

public class TinyHumansError : Exception
{
    public int Status { get; }
    public string Body { get; }

    public TinyHumansError(string message, int status, string body)
        : base(message)
    {
        Status = status;
        Body = body;
    }
}
